packages/code-studio/src/settings/EditorSectionContent.tsx

```diff
@@ -51,15 +51,24 @@ export function EditorSectionContent(): JSX.Element {
     (newValue: Record<string, unknown>) => {
       dispatch(
         updateNotebookSettings({
-          python: { linter: { ...ruffSettings, config: newValue } },
+          python: {
+            linter: {
+              ...ruffSettings,
+              config:
+                JSON.stringify(newValue) ===
```

Comment:
Is the argument mismatch in the JSON.stringify calls intentional?

Reply:
Yes. I want to check if the value was reset to default here, but the objects won't be the same. The other option was to do this check in the settings modal before onSave is called and then call with null or undefined to indicate the default.

Reply:
Oh, I misread this. I thought your were passing in `undefined, 2` to one of the JSON.stringify calls